discovery-aws-api/src/main/scala/akka/discovery/awsapi/ec2/Ec2TagBasedSimpleServiceDiscovery.scala

```diff
@@ -45,8 +45,6 @@ class Ec2TagBasedSimpleServiceDiscovery(system: ActorSystem) extends SimpleServi
       .withFilters(runningInstancesFilter)
       .withFilters(otherFilters)
 
-    implicit val timeout = resolveTimeout
```

Comment:
Wasn't being used, so removed to fix the compiler warning but also so as to not lead anyone into a false sense of security.